Gemfile

```diff
@@ -82,9 +93,10 @@ end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
-  gem 'capybara'
+  # gem 'capybara'
   gem 'ffi'
```

Comment:
Hello @KDlamini, thanks for the review ❤️ 
I do have the Capybara gem, but it was duplicated, so I commented one of them. I will implement the required changes.

Reply:
Hi @ZahraArshia,
Oops, my bad. Please remove the duplicate rather than comment it out. It's good practice to remove code that you do not need than to comment on it as per YAGNI principles.

Reply:
You are right, thanks for the tip 👍